cmake/FindDependencies.cmake

```diff
@@ -8,6 +8,7 @@ find_package(Sanitizers)
 
 ### Boost ###
 find_package(Boost REQUIRED COMPONENTS filesystem)
+find_package(Boost REQUIRED COMPONENTS program_options)
```

Comment:
`find_package` can take a list of components, so you can just add program_options to the previous call:
`find_package(Boost REQUIRED COMPONENTS filesystem program_options)`